demo/sdl2surface_rawfb/main.c

```diff
@@ -240,23 +260,25 @@ int main(int argc, char **argv)
         #endif
         /* ----------------------------------------- */
 
+        /*have Nuklear render to that surface now
         nk_sdlsurface_render(context, clear, 1);
-
-
-
-
+        
+        /* turn the surface into a texture: a format that a renderer can understand */
         tex = SDL_CreateTextureFromSurface(renderer, surface);
+        /* 'blit' the frame form the surface to the renderer for the gpu to draw */
         SDL_RenderCopy(renderer, tex, NULL, NULL);
+        /* show the renderer with its brand new frame */
         SDL_RenderPresent(renderer);
+        /* don't forget to free up that texture we just made! */
         SDL_DestroyTexture(tex);
 
     }
 
+    /* Have Nuklear shut down and clean up and all that */
     nk_sdlsurface_shutdown(context);
 
+    /* Have SDL do the same */
```

Comment:
"Do the same" is not very descriptive. While it makes sense in context, it could confuse readers.

``` suggestion
    /* Clean up and destroy all SDL resources */
```